.github/workflows/run-hibernate-orm-tests.yml

```diff
@@ -0,0 +1,49 @@
+name: Run Standard Integration Tests
```

Comment:
Can we change the name since we have another yml file with the same workflow name? eg Run Hibernate Integration Tests